sklearn/utils/sparsefuncs_fast.pyx

```diff
@@ -7,18 +7,21 @@
 # License: BSD 3 clause
 
 from libc.math cimport fabs, sqrt, isnan
+from libc.stdint cimport intptr_t
```

Comment:
Do we need this or better use our `intp_t` instead?

Reply:
I don’t think. I‘m confused and would prefer C/C++ over Cython, but that’s another story.

Reply:
Or can someone tell what the difference between `inp_t` alias `Py_ssize_t` and `intptr_t` alias `ssize_t` is?

Reply:
@jjerphan If this point is clear then maybe we can merge this PR?